content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -0,0 +1,163 @@
+selected, the option to specify that chosen field even further becomes available, via an
+:guilabel:`Add Filter` button.
+
+Clicking the :guilabel:`Add Filter` button, reveals fully customizable domain rule fields, which can
+be configured similar to an equation. You can create multiple recipient rules, if necessary. Then,
+Odoo will be sure to only send the SMS to recipients who fit into whatever criteria is configured in
+those fields.
+
+For instance, if :guilabel:`Contact` is chosen, all of the Odoo contacts in the database (vendors,
+customers, etc.) will receive the SMS, unless a specific recipient rule(s) are entered.
+
+In the example below, the message will only be sent to contacts who haven't chosen to
+:guilabel:`Blacklist` their contact information through the :guilabel:`Unsubscribing` process.
+
+.. image:: sms_essentials/contact-recipient.png
+   :align: center
+   :alt: Contact recipients on SMS marketing.
+
+How to create SMS messages
+==========================
+
+Enter the content of the SMS in the text field, found in the :guilabel:`SMS Content` tab. Links and
+emojis can also be included. Beneath the text field, Odoo displays how many characters are used in
+the message, along with how many SMS mailings it will take to deliver the complete message.
+
+.. tip::
+   To check the price of sending an SMS for a country, click on the :guilabel:`Information` icon.
+
+.. image:: sms_essentials/sms-price-check.png
+   :align: center
+   :alt: SMS price check icon
+
+.. note::
+   Credits must be purchased from Odoo in order to take advantage of the app (and its features).
+   SMS messages will not be sent without credits.
+
+.. seealso::
+   - :doc:`../pricing/pricing_and_faq`
+
+How to track links used in SMS messages
+---------------------------------------
+
+When links are used, Odoo automatically generates link trackers to gather analytical data and
+metrics related to those specific links, which can be found by going to
+:menuselection:`Configuration --> Link Tracker`.
+
+.. image:: sms_essentials/sms-link-tracker.png
+   :align: center
+   :alt: SMS Link Tracker page.
+
+How to adjust SMS Settings
+==========================
+
+Under the :guilabel:`Settings` tab of the SMS template, there is an option to :guilabel:`Include
+opt-out link`. If activated, the recipient is able to unsubscribe from the mailing list, thus
+avoiding all future mailings.
+
+An employee can be designated as the :guilabel:`Responsible` in the :guilabel:`Tracking` section of
+the :guilabel:`Settings` tab, as well.
+
+.. image:: sms_essentials/sms-settings-tab.png
+   :align: center
+   :alt: SMS Settings tab.
+
+How to send SMS messages
+========================
+
+Once a mailing is created, choose when Odoo should deliver the message from the following options:
+
+- :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
+  isn't excessive.
```

Comment:
Point of clarity: what does `excessive` mean here? How is this defined?

Consider the following suggestion, instead:
```suggestion
- :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
  is highly refined or in cases that involve fast approaching deadlines, such as a flash sale.
```